exec/java-exec/src/main/java/org/apache/drill/exec/store/pojo/PojoDataType.java

```diff
@@ -48,22 +51,25 @@ public PojoDataType(Class<?> pojoClass) {
       Class<?> type = f.getType();
       names.add(f.getName());
 
+      //Absence of annotation @Nonnull => (isNullable=true)
+      final boolean isNullable = f.isAnnotationPresent(Nonnull.class) ? false : true;
```

Comment:
This can be simplified to `! f.isAnnotationPresent(Nonnull.class)`.

Reply:
Agreed. Will do this change